<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoiEditor</class>
 <widget class="QWidget" name="RoiEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>269</width>
     <height>176</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="2">
     <widget class="QSpinBox" name="p1ySB">
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>-1000</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="copyButton">
      <property name="toolTip">
       <string>Copy ROI to clipboard</string>
      </property>
      <property name="text">
       <string>Copy</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QSpinBox" name="wySB">
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>-1000</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pxButton">
      <property name="toolTip">
       <string>Make the ROI 1 Pixel  wide along the integration axis of the ROI</string>
      </property>
      <property name="text">
       <string>1px</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="wxSB">
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>-1000</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>P2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="p1Label">
      <property name="text">
       <string>P1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="p1xSB">
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>-1000</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="widthLabel">
      <property name="text">
       <string>Width</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="p2xSB">
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>-1000</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QSpinBox" name="p2ySB">
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>-1000</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="hvCombo">
      <property name="toolTip">
       <string>Select if ROI should take a vertical or horizontal slice</string>
      </property>
      <item>
       <property name="text">
        <string>V</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>H</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pasteButton">
      <property name="toolTip">
       <string>Paste ROI from clipboard</string>
      </property>
      <property name="text">
       <string>Paste</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="stretchButton">
      <property name="toolTip">
       <string>Stetch the ROI to fill the entire image perpendicular to the integration axis of the ROI</string>
      </property>
      <property name="text">
       <string>Stretch</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="roiNameLabel">
      <property name="text">
       <string>ROI Name</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>p1xSB</tabstop>
  <tabstop>p1ySB</tabstop>
  <tabstop>p2xSB</tabstop>
  <tabstop>p2ySB</tabstop>
  <tabstop>wxSB</tabstop>
  <tabstop>wySB</tabstop>
  <tabstop>stretchButton</tabstop>
  <tabstop>pxButton</tabstop>
  <tabstop>copyButton</tabstop>
  <tabstop>pasteButton</tabstop>
  <tabstop>hvCombo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
